src/js/index.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+// test
+
+const $total = document.querySelector("#total");
+const $digits = document.querySelector(".digits");
+const $operations = document.querySelector(".operations");
+const $modifier = document.querySelector(".modifier");
+
+let isOperationClicked = false;
+let isOperator2In = false; // 두번째 인자가 들어왔는지 체크
+let operator1 = 0;
+let operator2 = 0;
+let operation = "";
+let result = 0;
+
+const initCalculator = () => {
```

Comment:
```suggestion
const resetCalculator = () => {
```

재설정이니 reset이란 이름이 더 좋을꺼 같습니다~!